site/docs/flink.md

```diff
@@ -44,8 +44,11 @@ To create iceberg table in flink, we recommend to use [Flink SQL Client](https:/
 Step.1 Downloading the flink 1.11.x binary package from the apache flink [download page](https://flink.apache.org/downloads.html). We now use scala 2.12 to archive the apache iceberg-flink-runtime jar, so it's recommended to use flink 1.11 bundled with scala 2.12.
 
 ```bash
-wget https://downloads.apache.org/flink/flink-1.11.1/flink-1.11.1-bin-scala_2.12.tgz
-tar xzvf flink-1.11.1-bin-scala_2.12.tgz
+FLINK_VERSION=1.11.1
+SCALA_VERSION=2.12
+APACHE_URL=https://downloads.apache.org
```

Comment:
Nit / non-blocking:

Have you tried running this script? Maybe you have cached dependencies, but I don't think Flink 1.11.1 is available at that URL anymore. 

One concern I have here is that this will only work for the few packages kept in the latest apache mirrors.

Currently, for Flink the main `downloads` URL has the latest Flink 1.11, 1.12, and 1.13 (i.e. 1.11.4, 1.12.5, and 1.13.2).

As you can see, this gets dated pretty quickly as the main `downloads` page is _only_ intended for the active latest major / minor versions (usually 2-3 major versions at a time, with only the latest minor version for each). This URL only has Flink 1.11.4.

I know this PR doesn't update the existing behavior, but arguably it gives the impression that this script can be used by simply changing the Flink version etc and that's not really the case.

The archives seem to have all versions, in addition to the current latest versions (from the current URL). The latest versions were pushed to the archive on the same day as they were pushed to the downloads link, so it appears that they're in sync.

Possibly it's better to instruct users to go to the archives instead? https://archive.apache.org/dist/flink/

In any case, if you can please ensure that you've run this script and that it works, that would be great.

Reply:
yes I verified it before submitting PR,. As you point out, that link was invalid later. Have replaced the URL with archives link.